.azure-pipelines/cleanup.yml

```diff
@@ -10,8 +10,5 @@ steps:
     if sudo [ -f dockerfs/var/run/docker.pid ] ; then
         pid=`sudo cat dockerfs/var/run/docker.pid` ; sudo kill $pid
     fi
-    sudo rm -rf dockerfs
-    sudo rm -rf fsroot
-    username=$(id -un)
-    sudo chown -R ${username}.${username} .
+    sudo rm -rf $(ls -A1)
```

Comment:
can you provide more description on what the issue was?

Reply:
@lguohan , for one reason, in PR build, we can not stop user to create files or folders with root permission, it will have issue when trying to checkout. For another reason, the local git repo may have some issue, when doing the merge/checkout multiple time.
The simple solution is to remove the entire folder including the hidden folders/files in the folder.